core/Cargo.toml

```diff
@@ -62,6 +62,7 @@ tokio = { version = "1.1", features = ["rt", "rt-multi-thread", "parking_lot", "
 tokio-util = { version = "0.7", features = ["io", "io-util"] }
 tokio-stream = "0.1"
 tonic = { version = "0.8", features = ["tls", "tls-roots"] }
+tower = { version="0.4.13" }
```

Comment:
Where do we use this?

Reply:
mod.rs:253
`tower::service_fn`

Afaik I can't just use some sort of a lambda for it